Towny/src/main/java/com/palmergames/bukkit/towny/object/PlayerCache.java

```diff
@@ -207,16 +204,17 @@ public boolean hasBlockErrMsg() {
 		return blockErrMsg != null;
 	}
 
+	@Deprecated

```

Comment:
Needs deprecated since version blah blah blah so we know when its safe to remove.